apps/judicial-system/backend/src/app/modules/court/court.service.ts

```diff
@@ -19,6 +21,10 @@ export const subTypes = {
   // 'Framsalsmál',
   // 'Frestur',
   CUSTODY: ['Gæsluvarðhald', 'Framlenging gæsluvarðhalds'],
+  ADMISSION_TO_FACILITY: [
```

Comment:
Looks like this subtype does not exist in Auður.

The existing subtypes are all listed in this file, some in comments as they are not yet part of RVG.

If I am correct, the value must be "Annað" and an extension is not possible.

Reply:
Should I change this to `ADMISSION_TO_FACILITY: ['Annað', 'Annað']` since this an extendable case?

Reply:
Not yet. First we should make sure it does not exist the Auður xRoad API.

Reply:
Ef við þurfum að nota Annað þá mundum við nota `ADMISSION_TO_FACILITY: 'Annað'`.